bridge/contracts/ValidatorPool.sol

```diff
@@ -255,6 +264,10 @@ contract ValidatorPool is
     }
 
     function minorSlash(address dishonestValidator_, address disputer_) public onlyETHDKG {
+        require(
```

Comment:
Adding more requirements to the slash functions (being a little bit more paranoic)